src/OpenApi/src/Schemas/OpenApiJsonSchema.Helpers.cs

```diff
@@ -290,7 +287,7 @@ public static void ReadProperty(ref Utf8JsonReader reader, string propertyName,
                 reader.Read();
```

Comment:
Interesting that we handle `anyOf` here but not `allOf` or `oneOf`. Is that because we know that the JSON we are reading comes from a source that never uses those keywords?

Reply:
Yes, this class is designed to convert the schemas we get from System.Text.Json to OpenApiSchemas. Currently, STJ only uses `anyOf` for polymorphic schemas.